sklearn/utils/metaestimators.py

```diff
@@ -111,7 +111,10 @@ def __get__(self, obj, owner=None):
                 )
 
         # lambda, but not partial, allows help() to work with update_wrapper
-        out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)  # noqa
+        if obj is None:
+            out = lambda *args, **kwargs: self.fn(*args, **kwargs)  # noqa
```

Comment:
For this branch, we need also do delegation attribute exist checking first (and raise AttributeError if delegation attribute does not exist), and then call `self.fn(*args, **kwargs)`

Reply:
e.g.:

```
pipeline_instance.predict_proba(X)
```
it will raise 'predict_proba' attribute not found error when `pipeline_instance._final_estimator` does not have `predict_proba` attribute.

Similarly, when we call:
```
Pipeline.predict_proba(pipeline_instance, X)
```
it should also do the delegation object attribute exist checking before really call `predict_proba` on delegated object.

Reply:
We can test this by:

```
import sklearn.svm
import sklearn.pipeline
estimators = [
    ("svc", sklearn.svm.SVC()),
]
pipeline_model = sklearn.pipeline.Pipeline(estimators)

pipeline_model.predict_proba(X) # raise attribute not found error
sklearn.pipeline.Pipeline.predict_proba(pipeline_model, X) # raise another error, behavior changed.
```